srcpkgs/swayimg/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'swayimg'
+pkgname=swayimg
+version=1.2
+revision=1
+build_style=meson
+hostmakedepends="pkg-config wayland-devel wayland-protocols"
```

Comment:
Huh, I wouldn't expect `wayland-protocols` to be necessary in host. Are you sure it's required?

Reply:
You're right, I just assumed it'd need both for arm builds. Removed wayland-protocols.